### PR TITLE
Increase zookeeper autoreconnect timer to 10s

### PR DESCRIPTION
**Pull request by forklady42**
Increasing the other sleep to 10s too

### CODE REVIEW

**Comment by colinmarc** [comment]
:+1:

**Comment by colinmarc** [comment]
Actually let's move this into a `const`.

**Comment by forklady42** [comment]
cool, will update

File: zk_watcher.go [hunk 1]
```diff
@@ -137,6 +137,8 @@ func (w *zkWatcher) cancelWatches() {
 	}
 }
 
+const waitPeriod = 10 * time.Second
```
Comment (colinmarc) [inline zk_watcher.go]: this is a const in the whole package, so it should be named more specifically. perhaps `zkReconnectPeriod` or similar?

Comment (forklady42) [inline zk_watcher.go]: great point! updated to `zkReconnectPeriod`

**Comment by colinmarc** [comment]
:+1: !!!

**Pull request merged**